v_4_4_0/master_template.go

```diff
@@ -544,9 +544,17 @@ storage:
     - path: {{ .Metadata.Path }}
       filesystem: root
       user:
-        name: {{ .Metadata.Owner.User }}
+      {{- if .Metadata.Owner.User.ID }}
+        id: {{ .Metadata.Owner.User.ID }}
+      {{- else }}
+        name: {{ .Metadata.Owner.User.Name }}
```

Comment:
Do you have a plan to get rid of Name at some point or we need to keep both?

Reply:
No, both of those params are supported in ignition spec, so I’d like to keep them (specially for cases like the one we have now in external-dns - same names in coreos on host system and alpine container have different id, so you can’t reference ownership with a name as then container user will not be able to get that data. So you need to use id of non-existing user on host